CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -413,6 +440,27 @@ private ContextMenu initColumnVisibilityContextMenu() {
         return cm;
     }
 
+    private void refreshColumnSelectionMenu(ArrayList<CustomMenuItem> columnCheckboxes, TextField columnFilterTextField, CheckBox columnCheckboxSource,
```

Comment:
If it doesn't break anything, it would be better for the `columnCheckboxes` to be a list rather than specifically an arraylist